lib/telemetry_test.go

```diff
@@ -0,0 +1,67 @@
+package lib
+
+import (
+	"errors"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/consul/logging"
+	"github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/require"
+)
+
+func newCfg() TelemetryConfig {
+	cfg := TelemetryConfig{
+		StatsdAddr:    "statsd.host:1234",
+		StatsiteAddr:  "statsite.host:1234",
+		DogstatsdAddr: "mydog.host:8125",
+	}
+	return cfg
```

Comment:
nit: you can just inline this.
```suggestion
	return TelemetryConfig{
		StatsdAddr:    "statsd.host:1234",
		StatsiteAddr:  "statsite.host:1234",
		DogstatsdAddr: "mydog.host:8125",
	}
```